server/src/test/java/org/folio/metastorage/matchkey/MatchKeyJsonPathTest.java

```diff
@@ -36,21 +36,21 @@ public void matchKeyJsonPathConfigureBad() {
     Exception e = Assert.assertThrows(
         MatchKeyException.class,
         () ->     matchKeyMethod.configure(jsonObject));
-    assertThat(e.getMessage(), is("jsonpath: either \"marc\" or \"inventory\" must be given"));
+    assertThat(e.getMessage(), is("jsonpath: expr must be given"));
   }
 
   @Test
   public void matchKeyJsonPathConfigureInvalidJsonPath() {
     MatchKeyMethod matchKeyMethod = new MatchKeyJsonPath();
-    JsonObject configuration = new JsonObject().put("marc", "$.fields.010.subfields[x");
+    JsonObject configuration = new JsonObject().put("expr", "$.fields.010.subfields[x");
     Assert.assertThrows(InvalidPathException.class,
         () -> matchKeyMethod.configure(configuration));
   }
 
   @Test
   public void matchKeyJsonPathConfigureMarc() {
     MatchKeyMethod matchKeyMethod = new MatchKeyJsonPath();
-    matchKeyMethod.configure(new JsonObject().put("marc", "$.fields.010.subfields[*].a"));
+    matchKeyMethod.configure(new JsonObject().put("expr", "$.fields.010.subfields[*].a"));
```

Comment:
Would it make sense to modify this test and wrap the Marc-in-JSON in {"marc": marc} so that the paths are confusing for someone reading the test?

Reply:
sure